examples/nlp/question-answering.py

```diff
@@ -0,0 +1,426 @@
+"""
+Title: Question Answering with Hugging Face Transformers
+Author: Matthew Carrigan and Merve Noyan
+Date created: 13/01/2022
+Last modified: 13/01/2022
+Description: Question answering implementation using Keras and Hugging Face Transformers.
+"""
+"""
+# Introduction to Question Answering
```

Comment:
All sections should use `##`, `#` is only for the main title (automatically added)